web-components/src/components/favorite/Favorite.ts

```diff
@@ -88,18 +87,20 @@ export namespace Favorite {
       return html`
         <label
           for="favorite-checkbox"
+          id="${this.id}"
           class="md-favorite ${classMap(this.favoriteClassMap)}"
-          tabindex="0">
-            <input
+          tabindex="0"
+        >
+          <input
             type="checkbox"
             aria-label=${ifDefined(this.label.length ? this.label : undefined)}
             value=${this.value}
             ?checked=${this.checked}
             ?disabled=${this.disabled}
             aria-hidden="true"
             name="favorite-checkbox"
-            />
-            <md-icon name="${this.getIconName()}" color="${this.checked ? "yellow" : nothing}"></md-icon>
+          />
+          <md-icon name="${this.getIconName()}" color="${this.checked ? "yellow" : nothing}"></md-icon>
         </label>
```

Comment:
Root cause is unknown, but I cannot toggle the Favorite star by using Space or Enter key. I can focus with Tab just fine, but cannot change its state.

Reply:
Fixed